config/locales/en.yml

```diff
@@ -74,8 +74,18 @@ en:
         someone_invited_you: "%{their_name} (%{their_email}) has invited %{your_name} to create an account on GetYourRefund to help people with their taxes."
         subject: You've been invited to GetYourRefund
   diy:
-    diy_intakes:
-      new:
+    continue_to_fsa:
+      edit:
+        continue_to_tax_slayer: Continue to TaxSlayer
```

Comment:
is there a separate story somewhere for "remove all instances of TaxSlayer from the copy"? because I thought the high level goal was to get that word off the website, not just out of the URL

Reply:
That's a great question. I **think** when Yvonne wrote this story I am working on, Yvonne assumed this page doesn't show up at all; probably **this** is the story where we decide if it's OK to say TaxSlayer on this page. I can take that question to Yvonne.

Reply:
@tgrathwell I checked with Yvonne, and she says on Slack:

> actually, don’t let this be a blocker. we have another story for the copy on that page. feel free to proceed!

https://cfa.slack.com/archives/GT1B45BS5/p1661445229317149